### PR TITLE
chore: refactored test login.spec.js

### PR DESCRIPTION
**Pull request by iamayushdas**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [ ] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches
- [x] Refactor

**What changes will this PR take into?**

Refactor Testcase for Frontend.

**Related issues**

related issue #1774 

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [x] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [x] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

**Review by juzhiyuan — APPROVED** [review]

File: web/cypress/integration/user/login.spec.js [hunk 1]
```diff
@@ -17,39 +17,45 @@
 /* eslint-disable no-undef */
 
 context('Login Test', () => {
+  const selector = {
+    errorExplain: '.ant-form-item-explain',
+    usernameInput: '#control-ref_username',
+    passwordInput: '#control-ref_password',
+    notification: '.ant-notification-notice-message',
+  };
+
```
[the remark this answers is not in the document]
Comment (iamayushdas) [reply web/cypress/integration/user/login.spec.js:26]: no, not here

[the remark this answers is not in the document]
Comment (liuxiran) [reply web/cypress/integration/user/login.spec.js:26]: but other prs define `data` to store test data,  it would be better to keep the same way

**Comment by iamayushdas** [comment]
@liuxiran can you retrigger CI , as i have checked it again, everything is working fine
Screenshot from 2021-05-11 21-09-12

**Comment by liuxiran** [comment]
> @liuxiran can you retrigger CI , as i have checked it again, everything is working fine
> Screenshot from 2021-05-11 21-09-12

Ok, retry CI. Mark the error: https://github.com/apache/apisix-dashboard/pull/1881/checks?check_run_id=2557014779, maybe it is an un stable case,need some time to fix it.

**Review by liuxiran — APPROVED** [review]

**Review by guoqqqi — APPROVED** [review]

**Review by LiteSun — APPROVED** [review]

**Pull request merged**